packages/augur-tools/src/libs/contract-api.ts

```diff
@@ -59,17 +74,10 @@ export class ContractAPI {
       feePerCashInAttoCash: new BigNumber(10).pow(16),
       affiliateFeeDivisor: new BigNumber(25),
       designatedReporter: this.account.publicKey,
-      extraInfo: JSON.stringify({categories: [" "], description: "description"}),
+      extraInfo: JSON.stringify({categories: [' '], description: 'description'}),
```

Comment:
might as well add categories for when Flash creates markets, we will want them to show up in the UI. 

I suggest `categories: ['flash', 'YesNo Market]`

Reply:
sure!